code/modules/mob/living/silicon/robot/drone/matriarchmaintdrone.dm

```diff
@@ -0,0 +1,25 @@
+/* /datum/ghostspawner/matriarchmaintdrone
```

Comment:
Why are we adding a completely commented out file?

Reply:
Also forgotten about. Deleted.